### PR TITLE
Undo #714 and use prow /lgtm and /approve

### PR DESCRIPTION
**Pull request by evankanderson**
See https://github.com/GoogleCloudPlatform/oss-test-infra/pull/714 and preceeding:

* https://github.com/GoogleCloudPlatform/oss-test-infra/pull/711
* https://github.com/GoogleCloudPlatform/oss-test-infra/pull/709 
* https://github.com/GoogleCloudPlatform/oss-test-infra/pull/702
* https://github.com/GoogleCloudPlatform/oss-test-infra/pull/659

This also carries over the "GitHub Review = LGTM + Approve" from this PR ([conversation](https://knative.slack.com/archives/CCSNR4FCH/p1610649354006500)):
* https://github.com/GoogleCloudPlatform/oss-test-infra/pull/667

Fixes https://github.com/knative/test-infra/issues/2752

### CODE REVIEW

**Comment by evankanderson** [comment]
(Need an `/lgtm`, too)

File: prow/knative/config.yaml [hunk 1]
```diff
@@ -137,6 +137,7 @@ tide:
   queries:
   - orgs:
     - "knative"
+    - "knative-sandbox"
     excludedRepos:
```
Comment (chizhg) [inline prow/knative/config.yaml:141]: Shall we also remove the excludedRepos?

**Review by chizhg — COMMENTED** [review]
/lgtm
/hold
In case you want to solve the comment.

File: prow/knative/config.yaml (hunk 1 above)
Comment (evankanderson) [reply prow/knative/config.yaml:141]: No, the `excludedRepos` are archived repos that won't accept any more PRs.

**Review by evankanderson — COMMENTED** [review]
/hold cancel

File: prow/knative/config.yaml (hunk 1 above)
Comment (evankanderson) [reply prow/knative/config.yaml:141]: I verified we don't have any archived repos in `knative-sandbox`

**Pull request merged**